pkg/apis/hub/v1alpha1/api_access.go

```diff
@@ -71,6 +79,7 @@ type APIAccessSpec struct {
 	OperationFilter *OperationFilter `json:"operationFilter,omitempty"`
 
 	// APIPlan defines which APIPlan will be used.
+	// +optional
 	APIPlan APIPlanReference `json:"apiPlan"`
```

Comment:
Could you add one more test on the APIAccess tests to check that we do allow empty APIPlans?